kotishell/src/tokenize/tokenize_a.c

```diff
@@ -48,7 +66,7 @@ static void	l_tk_end(t_tk_so_far *so_far, char *s)
 		l_tk_error_cleanup(so_far);
```

Comment:
also need a func to cleanup if ERR_SYMBOL

Reply:
yep!